pkg/cmd/root.go

```diff
@@ -30,6 +34,13 @@ func NewCmd() *cobra.Command {
 			"For detailed documentation please visit https://istio-workspace-docs.netlify.com/\n\n",
 		BashCompletionFunction: completion.BashCompletionFunc,
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			cmd.Flags().VisitAll(func(flag *pflag.Flag) {
+				if flag.Changed && strings.Join(flag.Annotations["silent"], "") == "true" {
```

Comment:
Can `flag.Annotations["silent"]` panic?

Reply:
No, it should be ok. You can read from an uninitialized map, but not write to one.